tests/NCronJob.Tests/NCronJobRetryTests.cs

```diff
@@ -89,50 +89,22 @@ public async Task JobShouldHonorJobCancellationDuringRetry()
     public async Task CancelledJobIsStillAValidExecution()
     {
         ServiceCollection.AddNCronJob(n => n.AddJob<CancelRetryingJob2>(p => p.WithCronExpression(Cron.AtEveryMinute)));
+
```

Comment:
@linkdotnet Although the diff noise may seem a tad scary, I believe I respected the semantics of the test